### ISSUE TITLE
“My” position not working

### ISSUE THREAD

**Issue by clickerclacker** [yenoiwesa/homebridge-connexoon#65]
Hi. I’ve set up the My position which works when using the TaHoma app, but when using the Home app to set the blinds to 50%, there is no response. I’ve left the config file in its default state. Hope this is an easy fix as it’s a great plugin!

**Comment by yenoiwesa** [comment]
Hello there @clickerclacker,

Could you share with me the `Connexoon` part of your Homebridge configuration JSON please, redacting credentials?

The **My** position should indeed be mapped by default as the 50% threshold 🤔 

Would also be interested in seeing debug logging from your Homebridge server when you select 50% for the window covering accessory in the Home app.

**Comment by clickerclacker** [comment]
HiThanks for the quick response!Here’s a couple of of screen shots for you.Cheers

**Comment by yenoiwesa** [comment]
@clickerclacker the screenshots didn't come through by email response.

**Comment by clickerclacker** [comment]
![776B6CFC-AF02-4227-948E-8A6B09FAA06B](https://user-images.githubusercontent.com/106246556/198901478-c53f1a50-e46a-40fd-92a1-a2b2242e67c9.jpeg)
![201120B3-7918-4745-95D2-EFB15930130B](https://user-images.githubusercontent.com/106246556/198901480-1f67babc-74ec-4b9b-a2e7-d7d214a7a8e3.jpeg)

**Comment by clickerclacker** [comment]
![590CB9F6-43EB-4551-9E03-CA1502C4496B](https://user-images.githubusercontent.com/106246556/198901666-1e409874-58e7-4335-a21d-5b1a151ed417.jpeg)
![FF53BA4B-B663-4B1D-8F7D-ACE6B00406DC](https://user-images.githubusercontent.com/106246556/198901671-7dd5c1bd-942d-4604-a9d9-3a876a3b0c7a.jpeg)

**Comment by yenoiwesa** [comment]
Thanks for the screenshots.

The logs show the Somfy servers return `UNSUPPORTED_OPERATION` for the command `my`. This is similar to this issue https://github.com/yenoiwesa/homebridge-connexoon/issues/28 where the user had not configured the device properly in the Somfy app, but I have no extra details about what was wrong they just closed the issue.

What type of device is your shade configured as in the Somfy app?

**Comment by clickerclacker** [comment]
The blinds are configured as Roller Blinds in the Somfy TaHoma app. The My button works on the Somfy app and the remote control. Any ideas welcome!

**Comment by yenoiwesa** [comment]
Sorry @clickerclacker I have actually no idea. Have you tried this plugin https://github.com/dubocr/homebridge-tahoma? Does it work with this one?

**Comment by yenoiwesa** [comment]
Hi @clickerclacker have you been able to check the other plugin?
I will be closing this issue tomorrow. Thanks.

**Comment by clickerclacker** [comment]
Hi. The homebridge-tahoma plug-in doesn’t seem to support the My position at all.

**Comment by davidliamfranks** [comment]
i have the same issue. 50% doesnt set the MY position. The other plugin doesnt support MY at all :(

**Comment by davidliamfranks** [comment]
<img width="787" alt="Screenshot 2022-12-31 at 19 26 10" src="https://user-images.githubusercontent.com/64067244/210153905-14ea20ea-47f4-4adb-a493-69be8cd31379.png">

could it be anything to do with the Device Type? This device type isnt in your list? but its a standard roller blind?

**Comment by yenoiwesa** [comment]
Hi @davidliamfranks thanks for the screenshot this is actually helpful. I see the model (which is mapped from the Somfy UI Widget in their API) is `DynamicScreen`. What did you choose in the Somfy RTS app to map your blind? Mine are all mapped as `UpDownScreen`, which allows the `My` command. I assume `DynamicScreen` does not allow the `My` command.

Could you show me your Somfy RTS app device list in the configuration tab?

**Comment by yenoiwesa** [comment]
Oh by the way guys your blinds are not RTS, they are `ogp` this is not the same protocol. This plugin only works with the Somfy RTS Connexoon!

**Comment by davidliamfranks** [comment]
I’m sure they are RTS? I thought it was just RTS and IO? Mine are defo RTS with an old fashioned RTS remote - in TaHoma is set them up as Window -> Somfy -> Window -> scan QR code on remote

My remote has up/my/down

The TaHoma box says it's HomeKit but only for IO blinds and mine don't appear natively for HomeKit so I'm sure it's just RTS
________________________________

**Comment by yenoiwesa** [comment]
@davidliamfranks I must confess I do not know what `ogp` really means in Somfy world. I saw online that you can have RTS `ogp` devices but how they differ to `rts` devices (with device URLs in `rts://`) I have no idea. All my devices' URL start with the protocol `rts://`.

I would be interested to get some details of the commands supported by your devices. Would you be able to navigate to your Homebridge's server `node_modules` directory and find the `homebridge-connexoon` directory in it.

Then, in that directory, please navigate to this path: https://github.com/yenoiwesa/homebridge-connexoon/blob/master/src/api/overkiz-api.js#L41 (`src/api/overkiz-api.js`)
and on line 41, add:

```js
this.log.info('Devices:', JSON.stringify(jsonDevices));
```

Then restart your Homebridge server, and paste here what you see for the new log entry from the plugin that starts with the `Devices:` prefix. It should look like a blob of text in JSON format.

I am particularly interested in the `commands` entries for your devices, to see what commands they support (since clearly `my` is not one of them).

Please let me know!

**Comment by davidliamfranks** [comment]
Hey, thanks for the quick reply!

this is the output....

```json
[{"creationTime":1671221255000,"lastUpdateTime":1671221255000,"label":"Bedroom Blind 1","deviceURL":"ogp://2101-4799-6320/13929830","shortcut":false,"controllableName":"ogp:Blind","definition":{"commands":[{"commandName":"close","nparams":0},{"commandName":"goToAlias","nparams":1},{"commandName":"identify","nparams":0},{"commandName":"open","nparams":0},{"commandName":"saveAlias","nparams":1},{"commandName":"setName","nparams":1},{"commandName":"setNodeDefinition","nparams":3},{"commandName":"stop","nparams":0}],"states":[{"type":"DiscreteState","values":["available","unavailable"],"qualifiedName":"core:AvailabilityState"},{"type":"DataState","qualifiedName":"core:NameState"},{"type":"DataState","qualifiedName":"core:RemovableState"}],"dataProperties":[],"widgetName":"DynamicScreen","uiProfiles":["OpenClose"],"uiClass":"Screen","qualifiedName":"ogp:Blind","type":"ACTUATOR"},"states":[{"name":"core:AvailabilityState","type":3,"value":"available"}],"attributes":[{"name":"core:Technology","type":3,"value":"rts"},{"name":"core:Manufacturer","type":3,"value":"Somfy"},{"name":"core:ManufacturerReference","type":3,"value":"blind"},{"name":"core:SubType","type":3,"value":"indoor"},{"name":"core:SupportedAlternativeNodeDefinitions","type":10,"value":[{"name":"blind","subTypes":[{"name":"indoor","definitions":["favorite1"]},{"name":"outdoor","definitions":["favorite1"]}]}]},{"name":"ogp:Features","type":10,"value":[{"name":"openClose","stateLess":true},{"name":"stop","stateLess":true},{"name":"identification"},{"name":"alternativeTyping"},{"name":"alias","stateLess":true}]},{"name":"core:SupportedAliases","type":10,"value":[{"id":"1","type":"favorite1","features":["openClose"]}]}],"available":true,"enabled":true,"placeOID":"ba384c3c-6813-4aef-99ea-f21db92b0b26","type":1,"widget":"DynamicScreen","oid":"00ad25e7-3498-40e0-9f6f-207c8a05abd6","uiClass":"Screen"},{"creationTime":1651605126000,"lastUpdateTime":1651605126000,"label":"Kitchen Blind","deviceURL":"ogp://2101-4799-6320/1525885","shortcut":false,"controllableName":"ogp:Blind","definition":{"commands":[{"commandName":"close","nparams":0},{"commandName":"goToAlias","nparams":1},{"commandName":"identify","nparams":0},{"commandName":"open","nparams":0},{"commandName":"saveAlias","nparams":1},{"commandName":"setName","nparams":1},{"commandName":"setNodeDefinition","nparams":3},{"commandName":"stop","nparams":0}],"states":[{"type":"DiscreteState","values":["available","unavailable"],"qualifiedName":"core:AvailabilityState"},{"type":"DataState","qualifiedName":"core:NameState"},{"type":"DataState","qualifiedName":"core:RemovableState"}],"dataProperties":[],"widgetName":"DynamicScreen","uiProfiles":["OpenClose"],"uiClass":"Screen","qualifiedName":"ogp:Blind","type":"ACTUATOR"},"states":[{"name":"core:AvailabilityState","type":3,"value":"available"}],"attributes":[{"name":"core:ManufacturerReference","type":3,"value":"blind"},{"name":"core:SupportedAlternativeNodeDefinitions","type":10,"value":[{"name":"blind","subTypes":[{"name":"indoor","definitions":["favorite1"]},{"name":"outdoor","definitions":["favorite1"]}]}]},{"name":"core:SupportedAliases","type":10,"value":[{"id":"1","type":"favorite1","features":["openClose"]}]},{"name":"core:Manufacturer","type":3,"value":"Somfy"},{"name":"ogp:Features","type":10,"value":[{"name":"openClose","stateLess":true},{"name":"stop","stateLess":true},{"name":"identification"},{"name":"alternativeTyping"},{"name":"alias","stateLess":true}]},{"name":"core:SubType","type":3,"value":"indoor"},{"name":"core:Technology","type":3,"value":"rts"}],"available":true,"enabled":true,"placeOID":"d1365e5e-70dd-4a14-8a0f-37703b5c4747","type":1,"widget":"DynamicScreen","oid":"ce7b7106-2480-4321-aca0-e4664e62934f","uiClass":"Screen"},{"creationTime":1650723824000,"lastUpdateTime":1650723824000,"label":"Connectivity kit-ledBox","deviceURL":"ogp://2101-4799-6320/15728639","shortcut":false,"controllableName":"ogp:Light","definition":{"commands":[{"commandName":"identify","nparams":0},{"commandName":"setIntensity","nparams":1},{"commandName":"setName","nparams":1}],"states":[{"type":"DiscreteState","values":["available","unavailable"],"qualifiedName":"core:AvailabilityState"},{"type":"ContinuousState","qualifiedName":"core:IntensityState"},{"type":"ContinuousState","qualifiedName":"core:LightIntensityState"},{"type":"DataState","qualifiedName":"core:NameState"},{"type":"DataState","qualifiedName":"core:RemovableState"}],"dataProperties":[],"widgetName":"DynamicLight","uiProfiles":["StatefulLightDimmer","StatefulDimmable","Dimmable"],"uiClass":"Light","qualifiedName":"ogp:Light","type":"ACTUATOR"},"states":[{"name":"core:NameState","type":3,"value":"Connectivity kit-ledBox"},{"name":"core:AvailabilityState","type":3,"value":"available"},{"name":"core:IntensityState","type":1,"value":100},{"name":"core:LightIntensityState","type":1,"value":100}],"attributes":[{"name":"core:SubType","type":3,"value":"status"},{"name":"core:ManufacturerReference","type":3,"value":"ledBox"},{"name":"ogp:Features","type":10,"value":[{"name":"dimming"},{"name":"identification"}]},{"name":"core:Technology","type":3,"value":"BoxTHmini"},{"name":"core:Manufacturer","type":3,"value":"Overkiz"}],"available":true,"enabled":true,"placeOID":"7d205775-a5cf-47d2-8bfe-01c4f576bd3d","type":1,"widget":"DynamicLight","oid":"6f674986-d581-453f-93f4-ba90e31b9e13","uiClass":"Light"},{"creationTime":1650723824000,"lastUpdateTime":1650723824000,"label":"io2way","deviceURL":"ogp://2101-4799-6320/16777212","shortcut":false,"controllableName":"ogp:Bridge","definition":{"commands":[{"commandName":"discover","nparams":0},{"commandName":"joinSetup","nparams":0},{"commandName":"sendPrivate","nparams":1},{"commandName":"shareSetup","nparams":0},{"commandName":"update","nparams":0}],"states":[{"type":"DiscreteState","values":["available","unavailable"],"qualifiedName":"core:AvailabilityState"},{"type":"DataState","qualifiedName":"core:FirmwareHashState"},{"type":"DataState","qualifiedName":"core:FirmwareRevisionState"},{"type":"DataState","qualifiedName":"core:NameState"},{"type":"DataState","qualifiedName":"core:NewFirmwareAvailableState"},{"type":"DataState","qualifiedName":"core:Private10State"},{"type":"DataState","qualifiedName":"core:Private1State"},{"type":"DataState","qualifiedName":"core:Private2State"},{"type":"DataState","qualifiedName":"core:Private3State"},{"type":"DataState","qualifiedName":"core:Private4State"},{"type":"DataState","qualifiedName":"core:Private5State"},{"type":"DataState","qualifiedName":"core:Private6State"},{"type":"DataState","qualifiedName":"core:Private7State"},{"type":"DataState","qualifiedName":"core:Private8State"},{"type":"DataState","qualifiedName":"core:Private9State"},{"type":"DataState","qualifiedName":"core:RemovableState"}],"dataProperties":[],"widgetName":"DynamicBridge","uiProfiles":["Specific"],"uiClass":"ProtocolGateway","qualifiedName":"ogp:Bridge","type":"ACTUATOR"},"states":[{"name":"core:NameState","type":3,"value":"io2way"},{"name":"core:AvailabilityState","type":3,"value":"available"},{"name":"core:FirmwareRevisionState","type":3,"value":"dev-43144AFB24B202BF012416491223CF57"},{"name":"core:FirmwareHashState","type":3,"value":"43144AFB24B202BF012416491223CF57"}],"attributes":[{"name":"core:ManufacturerReference","type":3,"value":"bridge"},{"name":"core:Technology","type":3,"value":"io2way"},{"name":"core:Manufacturer","type":3,"value":"Overkiz"},{"name":"ogp:Features","type":10,"value":[{"name":"discovery"},{"name":"setupManagement"},{"name":"firmware"},{"name":"private"},{"name":"identification","commandLess":true}]}],"available":true,"enabled":true,"placeOID":"7d205775-a5cf-47d2-8bfe-01c4f576bd3d","type":1,"widget":"DynamicBridge","oid":"296c3ecd-ce78-46b9-914d-cdb512961c19","uiClass":"ProtocolGateway"},{"creationTime":1650723824000,"lastUpdateTime":1650723824000,"label":"rts","deviceURL":"ogp://2101-4799-6320/16777214","shortcut":false,"controllableName":"ogp:Bridge","definition":{"commands":[{"commandName":"sendPrivate","nparams":1},{"commandName":"pair","nparams":3}],"states":[{"type":"DiscreteState","values":["available","unavailable"],"qualifiedName":"core:AvailabilityState"},{"type":"DataState","qualifiedName":"core:NameState"},{"type":"DataState","qualifiedName":"core:Private10State"},{"type":"DataState","qualifiedName":"core:Private1State"},{"type":"DataState","qualifiedName":"core:Private2State"},{"type":"DataState","qualifiedName":"core:Private3State"},{"type":"DataState","qualifiedName":"core:Private4State"},{"type":"DataState","qualifiedName":"core:Private5State"},{"type":"DataState","qualifiedName":"core:Private6State"},{"type":"DataState","qualifiedName":"core:Private7State"},{"type":"DataState","qualifiedName":"core:Private8State"},{"type":"DataState","qualifiedName":"core:Private9State"},{"type":"DataState","qualifiedName":"core:RemovableState"}],"dataProperties":[],"widgetName":"DynamicBridge","uiProfiles":["Specific"],"uiClass":"ProtocolGateway","qualifiedName":"ogp:Bridge","type":"ACTUATOR"},"states":[{"name":"core:NameState","type":3,"value":"rts"},{"name":"core:AvailabilityState","type":3,"value":"available"}],"attributes":[{"name":"core:Manufacturer","type":3,"value":"Overkiz"},{"name":"core:SupportedNodeDefinitions","type":10,"value":[{"name":"outlet","subTypes":[{"name":"default","definitions":["favorite1"]}]},{"name":"light","subTypes":[{"name":"default","definitions":["favorite1"]}]},{"name":"shutter","subTypes":[{"name":"rolling","definitions":["favorite1"]},{"name":"sliding","definitions":["favorite1"]},{"name":"swinging","definitions":["favorite1"]}]},{"name":"gate","subTypes":[{"name":"swinging","definitions":["pedestrian"]},{"name":"sliding","definitions":["pedestrian"]}]},{"name":"curtain","subTypes":[{"name":"single","definitions":["favorite1"]},{"name":"dual","definitions":["favorite1"]}]},{"name":"awning","subTypes":[{"name":"default","definitions":["favorite1"]}]},{"name":"blind","subTypes":[{"name":"indoor","definitions":["favorite1"]},{"name":"outdoor","definitions":["favorite1"]}]},{"name":"venetianBlind","subTypes":[{"name":"indoor","definitions":["favorite1"]},{"name":"outdoor","definitions":["favorite1"]},{"name":"indoor","definitions":["tiltOnlyWithFavorite1"]}]},{"name":"garageDoor","subTypes":[{"name":"rolling","definitions":["ventilation"]},{"name":"sliding","definitions":["pedestrian"]}]},{"name":"pergola","subTypes":[{"name":"default","definitions":["favorite1"]}]}]},{"name":"ogp:Features","type":10,"value":[{"name":"pairing"},{"name":"private"},{"name":"identification","commandLess":true}]},{"name":"core:ManufacturerReference","type":3,"value":"bridge"},{"name":"core:Technology","type":3,"value":"rts"}],"available":true,"enabled":true,"placeOID":"7d205775-a5cf-47d2-8bfe-01c4f576bd3d","type":1,"widget":"DynamicBridge","oid":"02a8f111-ac0f-461d-8e51-624eefc118d3","uiClass":"ProtocolGateway"},{"creationTime":1650723824000,"lastUpdateTime":1650723824000,"label":"Connectivity kit-box","deviceURL":"ogp://2101-4799-6320/16777215","shortcut":false,"controllableName":"ogp:Gateway","definition":{"commands":[{"commandName":"reboot","nparams":0},{"commandName":"reset","nparams":0},{"commandName":"sendPrivate","nparams":1},{"commandName":"setTimezone","nparams":1},{"commandName":"update","nparams":0}],"states":[{"type":"DiscreteState","values":["available","unavailable"],"qualifiedName":"core:AvailabilityState"},{"type":"DataState","qualifiedName":"core:ErrorState"},{"type":"DataState","qualifiedName":"core:ErrorsState"},{"type":"DataState","qualifiedName":"core:FirmwareHashState"},{"type":"DataState","qualifiedName":"core:FirmwareRevisionState"},{"eventBased":true,"type":"DataState","qualifiedName":"core:LocalAccessProofState"},{"type":"DataState","qualifiedName":"core:NameState"},{"type":"DataState","qualifiedName":"core:NewFirmwareAvailableState"},{"type":"DataState","qualifiedName":"core:Private10State"},{"type":"DataState","qualifiedName":"core:Private1State"},{"type":"DataState","qualifiedName":"core:Private2State"},{"type":"DataState","qualifiedName":"core:Private3State"},{"type":"DataState","qualifiedName":"core:Private4State"},{"type":"DataState","qualifiedName":"core:Private5State"},{"type":"DataState","qualifiedName":"core:Private6State"},{"type":"DataState","qualifiedName":"core:Private7State"},{"type":"DataState","qualifiedName":"core:Private8State"},{"type":"DataState","qualifiedName":"core:Private9State"},{"type":"DataState","qualifiedName":"core:RemovableState"},{"type":"DataState","qualifiedName":"core:SSIDState"},{"type":"ContinuousState","qualifiedName":"core:SignalQualityState"},{"type":"DataState","qualifiedName":"core:TimezoneState"}],"dataProperties":[],"widgetName":"DynamicGateway","uiProfiles":["Specific"],"uiClass":"ProtocolGateway","qualifiedName":"ogp:Gateway","type":"ACTUATOR"},"states":[{"name":"core:NameState","type":3,"value":"Connectivity kit-box"},{"name":"core:AvailabilityState","type":3,"value":"available"},{"name":"core:SignalQualityState","type":1,"value":100},{"name":"core:FirmwareRevisionState","type":3,"value":"v-2022.6.4-1-5e1f7a4"},{"name":"core:FirmwareHashState","type":3,"value":"2022.6.4-1:5e1f7a41237ee4819601934330b5f7ffd302127b:b6b68b6ed9ff16f859dc3d4f118dd430"},{"name":"core:SSIDState","type":3,"value":"WiFi"},{"name":"core:TimezoneState","type":3,"value":"Europe/Paris"}],"attributes":[{"name":"ogp:Features","type":10,"value":[{"name":"reset"},{"name":"localAccess"},{"name":"firmware"},{"name":"private"},{"name":"identification","commandLess":true},{"name":"diagnosis"},{"name":"supervision"},{"name":"time"},{"name":"wifi"},{"name":"networkv4"}]},{"name":"core:ManufacturerReference","type":3,"value":"gateway"},{"name":"core:Manufacturer","type":3,"value":"Overkiz"},{"name":"core:Technology","type":3,"value":"BoxTHmini"},{"name":"core:Private1","type":10,"value":[{"supportedPrivateCommands":"scheduledUpdate"}]}],"available":true,"enabled":true,"placeOID":"7d205775-a5cf-47d2-8bfe-01c4f576bd3d","type":1,"widget":"DynamicGateway","oid":"93e2a58f-1012-4a79-b553-8b5cfeb14a96","uiClass":"ProtocolGateway"},{"creationTime":1671222168000,"lastUpdateTime":1671222168000,"label":"Bedroom Blind 2","deviceURL":"ogp://2101-4799-6320/9095835","shortcut":false,"controllableName":"ogp:Blind","definition":{"commands":[{"commandName":"close","nparams":0},{"commandName":"goToAlias","nparams":1},{"commandName":"identify","nparams":0},{"commandName":"open","nparams":0},{"commandName":"saveAlias","nparams":1},{"commandName":"setName","nparams":1},{"commandName":"setNodeDefinition","nparams":3},{"commandName":"stop","nparams":0}],"states":[{"type":"DiscreteState","values":["available","unavailable"],"qualifiedName":"core:AvailabilityState"},{"type":"DataState","qualifiedName":"core:NameState"},{"type":"DataState","qualifiedName":"core:RemovableState"}],"dataProperties":[],"widgetName":"DynamicScreen","uiProfiles":["OpenClose"],"uiClass":"Screen","qualifiedName":"ogp:Blind","type":"ACTUATOR"},"states":[{"name":"core:AvailabilityState","type":3,"value":"available"}],"attributes":[{"name":"core:SubType","type":3,"value":"indoor"},{"name":"core:SupportedAlternativeNodeDefinitions","type":10,"value":[{"name":"blind","subTypes":[{"name":"indoor","definitions":["favorite1"]},{"name":"outdoor","definitions":["favorite1"]}]}]},{"name":"core:ManufacturerReference","type":3,"value":"blind"},{"name":"ogp:Features","type":10,"value":[{"name":"openClose","stateLess":true},{"name":"stop","stateLess":true},{"name":"identification"},{"name":"alternativeTyping"},{"name":"alias","stateLess":true}]},{"name":"core:Technology","type":3,"value":"rts"},{"name":"core:SupportedAliases","type":10,"value":[{"id":"1","type":"favorite1","features":["openClose"]}]},{"name":"core:Manufacturer","type":3,"value":"Somfy"}],"available":true,"enabled":true,"placeOID":"ba384c3c-6813-4aef-99ea-f21db92b0b26","type":1,"widget":"DynamicScreen","oid":"4b0c2f4d-b5df-4955-8d4a-8b73ee18c579","uiClass":"Screen"}]
```

**Comment by davidliamfranks** [comment]
could it be...

Open = Open
Close = Close
My = GoToAlias or favorite1 possibly?

would be really greatful if we could get this working, your implememntation seems perfect in HomeKit e.g. 3 positions (50% for MY) etc

**Comment by davidliamfranks** [comment]
I dont use Home Assistant but i just set it up quickly to try out their OverKiz integration and they managed to sort it for this Issue - https://github.com/home-assistant/core/issues/75067

So their implementation can access the MY button using favorite1 i think. BUT their implementation is weird as the MY button is in HA and HK as a seperate device so in HK the blind still just opens and closes but at least its proven the MY position can be accessed?

Your implementation on MY using 50% is much much better!

**Comment by yenoiwesa** [comment]
Thanks for the logs @davidliamfranks, yes it does look indeed that for some reason, Somfy decided to go with different command names for the `ogp` devices.

My RTS ones are:

```json
"commands": [
    { "commandName": "close", "nparams": 1 },
    { "commandName": "down", "nparams": 1 },
    { "commandName": "identify", "nparams": 0 },
    { "commandName": "my", "nparams": 1 },
    { "commandName": "open", "nparams": 1 },
    { "commandName": "rest", "nparams": 1 },
    { "commandName": "stop", "nparams": 1 },
    { "commandName": "test", "nparams": 0 },
    { "commandName": "up", "nparams": 1 },
    { "commandName": "openConfiguration", "nparams": 1 }
],
```
while yours are:
```json
"commands": [
    { "commandName": "close", "nparams": 0 },
    { "commandName": "goToAlias", "nparams": 1 },
    { "commandName": "identify", "nparams": 0 },
    { "commandName": "open", "nparams": 0 },
    { "commandName": "saveAlias", "nparams": 1 },
    { "commandName": "setName", "nparams": 1 },
    { "commandName": "setNodeDefinition", "nparams": 3 },
    { "commandName": "stop", "nparams": 0 }
],
```

So apparently I would indeed have to send the `goToAlias` command with the parameters array as  `['1']`. 

That should be doable but it's definitely not a 3 minute change. I will try to get something working tomorrow and will let you know.

Cheers!

**Comment by davidliamfranks** [comment]
Ah thanks, really appreciate your help!
________________________________

**Comment by yenoiwesa** [comment]
Actually I did a quick change, could you try this version of the plugin and let me know how it works? I have no means to test myself :-)

```
npm install -g "https://github.com/yenoiwesa/homebridge-connexoon.git#d23dd9f00f5a1399ee501bc8d41dcd5401a4ec88"
```

And restart your Homebridge server.

**Comment by davidliamfranks** [comment]
Yes that worked! Amazing, thank you so much!

**Comment by yenoiwesa** [comment]
Great news. Thanks for testing @davidliamfranks.

I released version `2.2.3` of the plugin which contains the change. Please upgrade to the final version.

Happy blind operations and happy new year!
